extensions/notion/src/components/CreateDatabaseForm.tsx

```diff
@@ -203,7 +203,7 @@ export function CreateDatabaseForm(props: { databaseId?: string; setRefreshView?
       const loadedDatabaseView = await loadDatabaseView(databaseId);
       const loadedDatabaseViewCopy = loadedDatabaseView ? JSON.parse(JSON.stringify(loadedDatabaseView)) : {};
 
-      if (loadedDatabaseView.create_properties) {
+      if (loadedDatabaseViewCopy?.create_properties) {
```

Comment:
that seems weird to me. If `loadedDatabaseViewCopy` can be undefined, then `loadedDatabaseViewCopy.create_properties = [];` on line 209 will crash as well.

Shall we change `const loadedDatabaseViewCopy = loadedDatabaseView ? JSON.parse(JSON.stringify(loadedDatabaseView)) : {};` to `const loadedDatabaseViewCopy = (JSON.parse(JSON.stringify(loadedDatabaseView)) : {}) || {};` to make sure it's never falsy?

Reply:
Hello @mathieudutour 👋 

I'm not sure to see in which case `loadedDatabaseViewCopy` can be undefined with this expression?
`const loadedDatabaseViewCopy = loadedDatabaseView ? JSON.parse(JSON.stringify(loadedDatabaseView)) : {}` 

It either has the value of `loadedDatabaseView`, which is for sure a `DatabaseView` object.
Or `{}`, an empty object that will be able to receive the `create_properties` attribute line 209:
`loadedDatabaseViewCopy.create_properties = [];`


If that's not the case (I'm not 100% a TypeScript expert), let me know and I'll change the code by what you suggested 😃

Reply:
I don't know either - but your change (adding a `?.`) is only useful if it can be.

Reply:
Ok got it, so I'll remove the `.?` as it shouldn't 🙏